website/docs/r/cloudfront_distribution.html.markdown

```diff
@@ -464,7 +461,7 @@ argument should not be specified.
 
 * `custom_origin_config` - The [CloudFront custom
     origin](#custom-origin-config-arguments) configuration information. If an S3
-    origin is required, use `s3_origin_config` instead.
+    origin is required, use `origin_access_control_id` or `s3_origin_config` instead.
```

Comment:
This wasn't clear - if you specify a `origin_access_control_id` - you don't need to give a `s3_origin_config` block.